doc/cookbook/source/examples/regression/least_square_regression.rst

```diff
@@ -0,0 +1,46 @@
+=======================
```

Comment:
Not so sure about this thing as we already have a cookbook for ridge regression

http://shogun.ml/examples/latest/examples/regression/linear_ridge_regression.html